index.php

```diff
@@ -154,6 +155,7 @@
 <script type="text/javascript" src="js/bootstrap-iconpicker.min.js"></script>
 <script type="text/javascript" src="js/main.js"></script>
 <script type="text/javascript" src="js/functions.js"></script>
+<script src="js/spectrum.js"></script>

```

Comment:
Declare type - some webservers do not properly send MIME-encoding on js files by default - add type="text/javascript"